testing/rabbitmq/APKBUILD

```diff
@@ -0,0 +1,91 @@
+# Contributor: Nathan Johnson <nathan@nathanjohnson.info>
+# Maintainer: Nathan Johnson <nathan@nathanjohnson.info>
+pkgname=rabbitmq
+_realname=rabbitmq-server
```

Comment:
I looked for other packages named -server and didn't find any.  I happen to like rabbitmq-server name better than rabbitmq personally